tests/fuzzer/fuzzer.py

```diff
@@ -179,6 +181,17 @@ def test_stdcm(
     """
     print("testing stdcm")
     stdcm_payload = make_stdcm_payload(scenario, path, rolling_stock)
+    r = post_with_timeout(editoast_url + f"infra/{scenario.infra}/load")
+    if r.status_code // 100 != 2:
+        if r.status_code // 100 == 4 and "no_path_found" in r.content.decode("utf-8"):
+            print("ignore: no path found")
+            return
+        make_error(
+            ErrorType.INFRA,
+            r,
+            infra_name,
+            {},
+        )
```

Comment:
THis shouldn't be done here but in the setup instead (around line 250). Otherwise the infra may not be loaded in pathfinding+schedule tests.